modules/wordads/wordads.php

```diff
@@ -180,9 +180,10 @@ function insert_head_meta() {
 		$themename = esc_js( get_stylesheet() );
 		$pagetype = intval( $this->params->get_page_type_ipw() );
 		$data_tags = ( $this->params->cloudflare ) ? ' data-cfasync="false"' : '';
+		$site_id = $this->params->blog_id;
```

Comment:
Is there ever a chance this would not be defined here?

Reply:
I asked other Derek about that before he left, but he did not think it was possible.